fineract-core/src/main/java/org/apache/fineract/portfolio/savings/SavingsAccountTransactionType.java

```diff
@@ -38,8 +38,9 @@ public enum SavingsAccountTransactionType {
     WITHDRAWAL_FEE(4, "savingsAccountTransactionType.withdrawalFee", TransactionEntryType.DEBIT), //
     ANNUAL_FEE(5, "savingsAccountTransactionType.annualFee", TransactionEntryType.DEBIT), //
     WAIVE_CHARGES(6, "savingsAccountTransactionType.waiveCharge"), //
-    PAY_CHARGE(7, "savingsAccountTransactionType.payCharge", TransactionEntryType.DEBIT), //
-    DIVIDEND_PAYOUT(8, "savingsAccountTransactionType.dividendPayout", TransactionEntryType.CREDIT), //
+    PAY_CHARGE(7, "savingsAccountTransactionType.payCharge"), //
```

Comment:
Please double check these modifications! The transaction entry types were deleted!

Reply:
Please double check as i dont see any changes...